docs/en/ingest-management/elastic-agent/install-agent-msi.asciidoc

```diff
@@ -10,13 +10,22 @@ The MSI package installer must be run by an administrator account. The installer
 
 . Download the latest Elastic Agent MSI binary from the link:https://www.elastic.co/downloads/elastic-agent[{agent} download page].
 
-. Run the installer:
+. Run the installer. The command varies slightly depending on whether you're using the default Windows command prompt or PowerShell.
++
+** Using the default command prompt:
 +
 [source,shell]
 ----
 elastic-agent-<VERSION>-windows-x86_64.msi INSTALLARGS="--url=<URL> --enrollment-token=<TOKEN>"
 ----
 +
+** Using PowerShell:
++
+[source,shell]
+----
+./elastic-agent-<VERSION>-windows-x86_64.msi --% INSTALLARGS="--url=<URL> --enrollment-token=<TOKEN>"
+----
++
 Where:
```

Comment:
the picky person in me is worried this looks like it applies to just the default command prompt. consider pulling out the bullets and making them all child paragraphs.

Reply:
@shainaraskas I've changed it to this, so the hierarchy is right. I'd like to avoid child paragraphs since it would differ from the formatting other, similar docs.

Here's how it looks now:

![Screenshot 2024-12-16 at 3 21 14 PM](https://github.com/user-attachments/assets/8ab58ca3-fd5d-4a3a-b114-e1ea0ab657f7)